### PR TITLE
Update to scala 2.12.4

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -20,7 +20,7 @@ object Dependencies {
   val aeronVersion = "1.3.0"
 
   val Versions = Seq(
-    crossScalaVersions := Seq("2.11.11", "2.12.3"),
+    crossScalaVersions := Seq("2.12.4", "2.11.11"),
```
Comment (patriknw) [inline project/Dependencies.scala]: Not sure that using 2.12 as default is good. Then there is a high risk that new things in 2.12 library is used and we don't find that until we run a later jenkins job with 2.11. What is the reason you want to make it default?

**Comment by ktoso** [comment]
I agree with using 2.12.4 as it's good for JDK9, but no need to make default I guess.
Well would be nice, but that carries a risk sadly (even just doing a trailing coma etc will cause errors then in the jenkins job "later on")

**Comment by ktoso** [comment]
I think jenkins, after upgrade of plugins comments twice... but one of the comments is somewhat nicer hm... Will try to figure out which one that is if I have the time tomorrow

**Review by patriknw — APPROVED** [review]
LGTM

**Review by ktoso — APPROVED** [review]

**Comment by ktoso** [comment]
Yay :)

**Pull request merged**